share/completions/funcsave.fish

```diff
@@ -1 +1,2 @@
 complete -c funcsave -xa "(functions -na)" -d "Save function"
+complete -c funcsave -s d -l directory -d "Directory to save the functions" -a $fish_function_path -r
```

Comment:
As it is, this will be expanded when the file is loaded, and expanded into multiple arguments *to complete*.

You need to quote it. If you double-quoted it, that would expand it when the file is loaded, as one argument, but it would always keep the value of $fish_function_path when the file was loaded.

So you need to *single*-quote it:

```fish
complete -c funcsave -s d -l directory -d "Directory to save the functions" -a '$fish_function_path' -r
```

This will cause *complete* to expand it when it needs to offer the completions.

Reply:
Done. Thank you for the explanation :)